sections/main-list-collections.liquid

```diff
@@ -14,14 +14,23 @@
     if section.settings.sort == 'products_high' or section.settings.sort == 'date_reversed' or section.settings.sort == 'alphabetical_reversed'
       assign collections = collections | reverse
     endif
+
+    assign moduloResult = 28 | modulo: section.settings.columns_desktop
+    assign paginate_by = 30
+    if moduloResult == 0
+      assign paginate_by = 28
+    endif
```

Comment:
Can you share more on where are these `28`/`30` numbers are coming from? 

Did you base this on an existing theme (or part of our theme) or is there any UX docs suggesting these numbers?

If I look at collection page's default:
- Default columns: 4 (smaller cards)
- Default items per page: 16
- Rows: 4

But here we have:
- Default columns: 3 (larger cards)
- Default items per page: 30
- Rows: 10

Anyway, this PR is a big improvement over having no pagination and we have #1767 to add more flexibility to this section, but I'm curious if there was any exploration on this temporary number we're shipping by default  :D

Reply:
Yeah, the logic makes sense to me. I was wondering why we went with `28`/`30` and not `14`/`15` or another number that better matches the default state for other sections such as collection pages :D

I'm fine with it as is and then expanding with the settings Meli suggested :D